src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -155,15 +155,17 @@
         </dict>
       </dict>
 
-      <dict>
+      <!-- Temporarily removed because IN VS TextMate sometimes superceeds Semantic tokes
```

Comment:
We turn this off in TextMate to avoid the following problem:
![](https://user-images.githubusercontent.com/14987221/103803822-f7f4b480-5005-11eb-84b7-e00ccac5076c.png)

Reply:
Could you share some screenshots to show what it looks like now?